blocks/footer/footer.js

```diff
@@ -51,10 +51,16 @@ function decorateFooter(block) {
  */
 export default async function decorate(block) {
   block.textContent = '';
+  let footerPath = 0;
+  let folder = 0;
 
   // fetch footer content
   const footerMeta = getMetadata('footer');
-  const footerPath = footerMeta || (getLanguage() === 'jp' ? '/footer' : `/${getLanguage()}/footer`);
+  const action = getMetadata('template');
```

Comment:
I'd rather make this more generic. The metadata can be called `page-type` and a value of `dentistry` should result in the dentistry related header/footer to appear.

Reply:
@sdmcraft  - Thank you for the feedback. Actually I was trying to make things easier for the customer by just leveraging metadata.xlsx file. I have used the below - 

<img width="474" alt="image" src="https://github.com/hlxsites/sunstar-foundation/assets/37831850/05d2e84f-8d82-487a-bb70-e0e8961cf276">

I would rather prefer the customer to just update the metadata sheet instead of going specially into the Code & append any other directory for any unique customisation later on. If tomorrow they want to add another set of directory for a unique header / footer then they have to just mention - customnavfooter-directory in the sheet, that's it. The logic will automatically change the footer / header for that set of directory. Plus not only applying a custom nav footer functionality but any other functionality they would like to apply to any particular directory - they can simply add - 

functionality-dirdectoyname in the sheet. And then add the same logic wherever they want. Please advise.

Reply:
Yes, the idea to use `metadata.xlsx` is great. I am just suggesting to rename customnavfooter-dentistry` to just `dentistry`. Reasons - It controls not only footer but header also. Secondly it keeps option open to use it for other purposes in future.

Reply:
Sure. Updated the sheet and JS logic as well accordingly. Thank You.